package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-sqlcipher-adapter",
-  "version": "0.1.7-os.3",
+  "version": "0.1.7-OS3",
```

Comment:
This is affecting the current (non semver compliant) version but before making a release. Is this ok with the team processes in place?

Reply:
Yup, actually I just changed this to normalize the plugin versions with the versions we use in all our other supported plugins 😄

Reply:
The current meta is to use this convention for forked repositories: `<upstream_tag>-OSX` where `X` is the number of custom patches that have been applied to the code. The idea is to not deviate from the upstream release tag.